usage2/SQLiteWrapperLanguage.md

```diff
@@ -269,7 +269,7 @@ Delete.tables(MyTable1.class, MyTable2.class);
 
 // Delete using query
 SQLite.delete(MyTable.class)
-  .where(DeviceObject_Table.carrier.is("T-MOBILE"))
+  .where(DeviceObject_Table.carrier.is("TModel-MOBILE"))
```

Comment:
#req looks like you did some kind of refactor that change T to TModel. haha oops.